client/src/app/site/agenda/components/agenda-import-list/agenda-import-list.component.ts

```diff
@@ -4,12 +4,12 @@ import { MatSnackBar } from '@angular/material';
 import { Title } from '@angular/platform-browser';
 import { TranslateService } from '@ngx-translate/core';
 
-import { AgendaImportService } from '../../agenda-import.service';
+import { AgendaImportService } from '../../services/agenda-import.service';
 import { BaseImportListComponent } from 'app/site/base/base-import-list';
 import { CsvExportService } from 'app/core/ui-services/csv-export.service';
 import { DurationService } from 'app/core/ui-services/duration.service';
 import { itemVisibilityChoices } from 'app/shared/models/agenda/item';
-import { ViewCreateTopic } from '../../models/view-create-topic';
+import { ViewCreateTopic } from '../../../topics/models/view-create-topic';
```

Comment:
seems like a very long path. Urually, the importer would try to find the shortest path,
`app/site/topics` seems more intuitive than going three levels up.

Try Re-Importing and/or try to change the following settings in VSCode:

`Javascript -> Preferences -> Module Import Specifier'